src/Appwrite/Utopia/Response/Model/Webhook.php

```diff
@@ -76,7 +76,18 @@ public function __construct()
                 'default' => '',
                 'example' => 'ad3d581ca230e2b7059c545e5a',
             ])
-        ;
+            ->addRule('enabled', [
+                'type' => self::TYPE_BOOLEAN,
+                'description' => 'Indicates if this webhook is enabled.',
+                'default' => true,
+                'example' => true,
+            ])
+            ->addRule('logs', [
```

Comment:
Attempts are not made public. Just making sure, is this on purpose? I don't see downside of allowing developers to see how many attempts were made.